src/formatDate.js

```diff
@@ -51,6 +51,38 @@
 
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const fromFormatObj = {};
+  const fromF = fromFormat[3];
+  const toF = toFormat[3];
+  const dateArr = date.split(fromF);
```

Comment:
arr = tepe, don't use types as names if you need to call variable with array use plural form